WireDomain/Sources/WireDomain/Repositories/ConversationsLabels/ConversationLabelsLocalStore.swift

```diff
@@ -22,20 +22,16 @@ import WireLogging
 // sourcery: AutoMockable
 public protocol ConversationLabelsLocalStoreProtocol {
 
-    /// Save label and related conversations objects to local storage.
-    /// - Parameter conversationLabel: conversation label info
+    /// Stores the label and updates conversation relationships in
+    /// the local storage.
+    ///
+    /// After this method is invoked, only the given labels exist
+    /// in the local storage, i.e all old labels will be deleted.
+    ///
+    /// - Parameter labels: The labels to store.
 
-    func storeLabel(
-        _ conversationLabel: ConversationLabelInfo
-    ) async throws
-
-    /// Delete old `folder` labels and related conversations objects from local storage.
-    /// - Parameter excludedLabels: remote labels that should be excluded from deletion.
-    /// - Only old labels of type `folder` are deleted, `favorite` labels always remain in the local storage.
+    func setLabels(_ labels: [ConversationLabelInfo]) async throws
 
-    func deleteOldLabelsLocally(
-        excludedLabels: [ConversationLabelInfo]
-    ) async throws
```

Comment:
We were only calling these methods in conjunction, so I replaced them with a single `setLabels` method.